File: Otasukejuru/app/src/main/java/jp/ict/muffin/otasukejuru/CustomCell.java
package jp.ict.muffin.otasukejuru;

import android.content.Context;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.TextView;


public class CustomCell extends LinearLayout{
    private TextView date;
    private TextView viewTask1;
    private TextView viewTask2;
    private TextView viewTask3;

    public CustomCell(Context context, AttributeSet attr) {
        super(context, attr);

        View layout = LayoutInflater.from(context).inflate(R.layout.cell, this);

        viewTask1 = (TextView)layout.findViewById(R.id.task_view1);
        viewTask2 = (TextView)layout.findViewById(R.id.task_view2);
        viewTask3 = (TextView)layout.findViewById(R.id.task_view3);
    }

}

File: Otasukejuru/app/src/main/kotlin/jp/ict/muffin/otasukejuru/view/CustomCell.kt
package jp.ict.muffin.otasukejuru.view

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import android.widget.LinearLayout
import android.widget.TextView
import jp.ict.muffin.otasukejuru.R


class CustomCell(context: Context, attr: AttributeSet) : LinearLayout(context, attr) {
    private val date: TextView? = null
    private val viewTask1: TextView
    private val viewTask2: TextView
    private val viewTask3: TextView
    
    init {
        
        val layout = LayoutInflater.from(context).inflate(R.layout.cell, this)
        
        viewTask1 = layout.findViewById(R.id.task_view1) as TextView
        viewTask2 = layout.findViewById(R.id.task_view2) as TextView
        viewTask3 = layout.findViewById(R.id.task_view3) as TextView
    }
    
}
